tests/resources/import/ids.json

```diff
@@ -1,11 +1,13 @@
-[
-  {
-    "id": "00001"
-  },
-  {
-    "id": 11586
-  },
-  {
-    "id": 3
-  }
-]
\ No newline at end of file
+{
+  "songs": [
+    {
+      "id": "00001"
+    },
+    {
+      "id": 11586
+    },
+    {
+      "id": 3
+    }
+  ]
+}
```

Comment:
These don't match the format you specified. In your comment and the Python implementation they are nullable, i.e. the key must exist, but the value may be null. Here, it seems as if they are optional, i.e. not even the key is required.

Reply:
I just wanted to test the bare minimum as the import ignores all other fields. - should I use the exact export format here?

Reply:
The purpose of the export format is to export all available information, the purpose of the import format is to define a couple of ids for import, where data and infos are retrieved from USDB. Within this PR it seemed weird if the import JSON format is totally incompatible with the export JSON Format. So I changed this that an exported JSON may be imported again, as well as self-generated JSON just from a couple of ids in proper way.

Reply:
Would be odd if the fixtures didn't match the format. But it's your format, you can just decide that these field are optional. 🙂 Only then, they shouldn't also be nullable.

Reply:
I don't see the point of making the JSON import format identical with the JSON export format, my goal was to make the export format compatible with the import format, not the other way round:

1. If this import format would not be valid, one that just wants to import a couple of usdb_ids from USDB would be force to use the `.usdb_ids` file or would have populate the JSON with extra fields that have no specific purpose and would not need to be correct for the import to work.
2. In addition if I added and checked the extra fields in the import format I would actually also need to check e.g. the `title` and `artist` to match USDB and inform the user if they changed or do some other action if I see changes.
3. Finally a user might not know the correct artist and title and just have a list of ids to import. The user could just wild guess titles and artists and the import would still work or fail if not guessed correctly.

Do we really want to validate the JSON format on import match the JSON format on export?

Reply:
> I don't see the point of making the JSON import format identical with the JSON export format

I agree, there's no reason the ID JSON *had* to be the same as the song JSON. But since you've deliberately changed the fixtures, I thought that would be your plan.